app/views/tcsp/ServicesOfAnotherTCSPView.scala.html

```diff
@@ -44,13 +44,7 @@
 
     <p class="govuk-body">
         @messages("tcsp.servicesOfAnotherTcsp.hint.1")
-        @hmrcNewTabLink(
-            NewTabLink(
-                classList = Some("govuk-link"),
-                href = Some(appConfig.tcspWhoNeedsToRegisterLink),
-                text = messages("tcsp.servicesOfAnotherTcsp.hint.1.link.text")
-            )
-        )
+        <a href = "@appConfig.tcspWhoNeedsToRegisterLink"> @messages("tcsp.servicesOfAnotherTcsp.hint.1.link.text")</a>
```

Comment:
Link is the wrong shade of blue, just need to add the `govuk-link` class to the a tag
https://design-system.service.gov.uk/styles/links/
<img width="733" alt="Screenshot 2024-04-12 at 11 28 16" src="https://github.com/hmrc/amls-frontend/assets/60218795/acf20e64-e409-4306-8c8b-6558556c1933">